examples/for-tests-react-16/src/testContext.js

```diff
@@ -9,11 +9,17 @@ export function getTestContext() {
         thirdPartyRedirectURL: localStorage.getItem("thirdPartyRedirectURL"),
         authRecipe: window.localStorage.getItem("authRecipe") || "emailpassword",
         usesDynamicLoginMethods: localStorage.getItem("usesDynamicLoginMethods") === "true",
+        enableAllRecipes: localStorage.getItem("enableAllRecipes") === "true",
         clientRecipeListForDynamicLogin: localStorage.getItem("clientRecipeListForDynamicLogin"),
         mockLoginMethodsForDynamicLogin: localStorage.getItem("mockLoginMethodsForDynamicLogin"),
         staticProviderList: localStorage.getItem("staticProviderList"),
         mockTenantId: localStorage.getItem("mockTenantId"),
         clientType: localStorage.getItem("clientType") || undefined,
```

Comment:
not related to this file, but the icon for TOTP in the factor choosers screen needs to change

Reply:
Keeping this open since this is not yet done